circleci/docker-publish

```diff
@@ -44,9 +46,11 @@ SHORT_SHA=${CIRCLE_SHA1:0:7}
 
 DOCKER_USER=$1
 DOCKER_PASS=$2
-DOCKER_EMAIL=$3
 ORG=$4
```

Comment:
$3 ?

Reply:
During this change, I don't plan on updating all the other repos (I'd have to update their calls to remove docker email) ... So I'm just ignoring that arg

Reply:
Oh I see now. makes sense